### ISSUE TITLE
process_traces's metric mode is confusing

### ISSUE THREAD

**Issue by eakuefner** [catapult-project/catapult#4112]
People should just use `tracing/bin/run_metric` if they want to run metrics on traces. We should paper over process_traces to recommend that people do that instead. We could even just go ahead and build the command line that people should use, and print that.

@simonhatch

**Comment by maxlgu** [comment]
Here are some docs describing how to use slow_reports/process_traces to process traces. Should they also be updated as well?
https://docs.google.com/document/d/1tL8HfU5LzJOX7T_KCZJD29uzG5FgNtYNp54_I2HiRqU/edit
https://docs.google.com/document/d/1wnTpTsA9ZpuY6fGH3iq4eHAfeNKVk4woVWmMxYexR6Y/edit

@deepanjanroy

**Comment by deepanjanroy** [comment]
I think the slow_reports/process_traces in google3 is different from the [process_traces.py](https://github.com/catapult-project/catapult/blob/master/trace_processor/trace_processor/process_traces.py) Ethan is talking about here. He's right - for locally running a metric `run_metric` is a lot better than using `process_traces.py`. 

@eakuefner  process_traces.py is still useful when we want to write a mapper instead of a metric. I'm guessing that use case will stay?

**Comment by eakuefner** [comment]
Yes, I think process_traces' mapper functionality can stay for the time being. The main bug I'm concerned about here is that we have two ways of running metrics in Catapult, one strictly worse.
